src/azsdkdemoprocessor/src/main/java/demo/sdk/az/App.java

```diff
@@ -0,0 +1,93 @@
+package demo.sdk.az;
+
+import ch.qos.logback.classic.Logger;
+import com.azure.identity.*;
+import com.azure.storage.queue.*;
+import com.microsoft.applicationinsights.logback.ApplicationInsightsAppender;
+import io.github.cdimascio.dotenv.*;
+import java.util.*;
+import org.slf4j.LoggerFactory;
+
+public class App {
+  static final Logger logger = (Logger) LoggerFactory.getLogger("root");
```

Comment:
```suggestion
  private static final Logger LOGGER = LoggerFactory.getLogger(App.class);
```

Reply:
why LOGGER?

I wasn't able to find the aiAppender when I used App.class, so I changed to root and it worked.

Reply:
okay, looks like consts are upper in java as a convention.

do you know why App.class wouldn't work?  Something to do with my logback.xml file?